routing/missioncontrol.go

```diff
@@ -231,7 +231,13 @@ func (p *paymentSession) RequestRoute(payment *LightningPayment,
 	// Taking into account this prune view, we'll attempt to locate a path
 	// to our destination, respecting the recommendations from
 	// missionControl.
-	path, err := findPath(nil, p.mc.graph, p.mc.selfNode, payment.Target,
+
+	g, err := newMemChannelGraphFromDatabase(p.mc.graph)
```

Comment:
Have you benchmarked this change? I suspect this will be quite slow, as you will need to read the while database before every call to `findPath`

Reply:
I’ll try to benchmark it, however I can’t imagine it to be much slower asymptotically since findPath itself is O((N+M)*logM)

Reply:
Maybe you're right. This will read the whole db (instead of only the vertices/edges  needed to find the path), but maybe it's not that slow, since it will batch all reads in one.

A better approach would be to read the db into memory once, though, and reuse it in each call to `RequestRoute`.

Reply:
Right, I realised that, and will make a commit to replace `missioncontrol.g`(mission control's graph) by an in memory version instead, I think that has many merits otherwise too.